src/kit.ts

```diff
@@ -200,10 +200,16 @@ export async function kitIfCompiler(bin: string, pr?: ProgressReporter): MaybeCo
  * @param dir Directory containing candidate binaries
  * @returns A list of CompilerKits found
  */
-export async function scanDirForCompilerKits(dir: string, pr?: ProgressReporter) {
+export async function scanDirForCompilerKits(dir: string, pr?: ProgressReporter) : Promise<Kit[]> {
+  if (! await fs.exists(dir)) {
+    log.debug('Skipping scan of not existing path', dir);
+    return [];
```

Comment:
I think this should be returned as a `Promise`:
```ts
return return Promise.resolve([]);
```

Reply:
Style guide question